codec-http/src/main/java/io/netty/handler/codec/http/HttpUtil.java

```diff
@@ -381,6 +383,29 @@ public static CharSequence getCharsetAsString(HttpMessage message) {
         return null;
     }
 
+    /**
+     * Fetch MIME type part from message's Content-Type header as a char sequence.
+     *
+     * @return the MIME type as a {@code CharSequence} from message's Content-Type header
+     * or {@code null} if content-type header or MIME type part of this header are not presented
+     * <p/>
+     * "content-type: text/html; charset=utf-8" - "text/html" will be returned <br/>
+     * "content-type: text/html" - "text/html" will be returned <br/>
+     * "content-type: " or no header - {@code null} we be returned
+     */
+    public static CharSequence getMimeType(HttpMessage message) {
```

Comment:
`getContentType` should return full content-type header's content I think, not just mime-part of content-type header. So, it could be misleading name. WDYT?